data/xml/2024.luhme.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<collection id="2024.luhme">
+  <volume id="1" ingest-date="2025-01-09" type="proceedings">
+    <meta>
+      <booktitle>Proceedings of the First LUHME Workshop</booktitle>
+      <editor><first>Rui Sousa</first><last>Silva</last></editor>
+      <editor><first>Henrique Lopes</first><last>Cardoso</last></editor>
+      <editor><first>Maarit</first><last>Koponen</last></editor>
+      <editor><first>Antonio Pareja</first><last>Lora</last></editor>
+      <editor><first>M\'{a}rta</first><last>Seresi</last></editor>
```

Comment:
bibtex parse failure

Reply:
ideally you would add something to the script to look for `\'` and/or `{` in names, which should never be valid, and would flag these